zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/processinstance/CreateProcessInstanceRejectionTest.java

```diff
@@ -298,7 +298,7 @@ public void shouldRejectCommandIfTagsAreInvalid() {
         .hasRejectionType(RejectionType.INVALID_ARGUMENT)
         .hasRejectionReason(
             String.format(
-                "Expected to create instance of process with tags, but tag '%s' is invalid. Tag must start with a letter (a-z, A-Z), followed by alphanumerics, underscores, minuses, colons, or periods. It must not be blank and must be 100 characters or less.",
+                "Expected to create instance of process with tags, but the tags '%s' are invalid. Tags must start with a letter (a-z, A-Z), followed by alphanumerics, underscores, minuses, colons, or periods. It must not be blank and must be 100 characters or less.",
```

Comment:
🔧 We should add also that the max number of tags is 10

Reply:
There is another test and another rejection reason when the limit is exceeded. https://github.com/camunda/camunda/pull/36873/files/a15afd12a7e6f92f34e81700d838095212cba9e2..af939528554d3fa51b9db2f3be85b4ccf4b40496#diff-1a9c01c3ce090f6e0a888f8ee808fd5a5f7ba5737cf057e2b80a2109e24b1722R306